contracts/forge-test/NFTReward_Unit.t.sol

```diff
@@ -264,8 +267,8 @@ contract TestJBTieredNFTRewardDelegate is Test {
 
     _delegate.transferOwnership(owner);
 
-    assertTrue(_isIn(_delegate.test_store().tiers(address(_delegate), 0, numberOfTiers), _tiers));
-    assertTrue(_isIn(_tiers, _delegate.test_store().tiers(address(_delegate), 0, numberOfTiers)));
+    assertTrue(_isIn(_delegate.test_store().tiers(address(_delegate), 0, 0, numberOfTiers), _tiers));
```

Comment:
can't we test the catergory too (instead of 0)?

Reply:
We have a separate test for that to fetch a particular category check the last commit